src/App.tsx

```diff
@@ -152,11 +163,36 @@ class Board extends React.Component<BoardProps, BoardState> {
         this.setState(newState);
     }
 
-    public handlePress = (e: KeyboardEvent) => {
-        let i = parseInt(e.key, 10);
-        console.log(i);
+    /**
+     * deselect / unhighlight all highlighted squares
+     */
+    public deselectSquares(): void {
+        const newState = _.cloneDeep(this.state);
+        newState.highlighting = false;
+        newState.highlights = new Set<SquareAddress>();
+        this.setState(newState);
+    }
+
+    public handleGlobalKeyDown = (e: KeyboardEvent) => {
+        if (e.keyCode === 8) { // Delete Key
+            e.preventDefault();
+            this.deleteSelectedSquares();
+            return;
+        }
+        if (e.keyCode === 9) { // Tab Key
+            e.preventDefault();
+            this.toggleNumpadMode();
+            return;
+        }
+        if (e.keyCode === 91) { // Command Key
```

Comment:
This is showing up as `[` for me. Are you sure it's command?

Reply:
yeah `[` shows up at 219 for me